gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-details/api-tab-details.component.scss

```diff
@@ -38,3 +38,20 @@
     }
   }
 }
+
+.info__miscellaneous {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.info__miscellaneous_item {
+  padding: 1px 0;
+  line-height: 22px;
+  vertical-align: middle;
+}
+
+.info__miscellaneous_item span {
+  padding: 1px 5px 1px 0;
+  opacity: 0.6;
+}
```

Comment:
🗑️  -> Reuse the styles that are already in this file

Reply:
Done